src/screens/Profile/Profile.js

```diff
@@ -257,9 +257,9 @@ class Profile extends React.Component<Props, State> {
     this.setState((prev: State) => ({ showTrackingModal: !prev.showTrackingModal }));
   }
 
-  handleChangeUseBiometrics = (value) => {
+  handleChangeUseBiometrics = (value, privateKey) => {
```

Comment:
Could we rename `value` to something else? As it means nothing

Reply:
Sure 👍

Reply:
Updated in `Settings.js` also per PR note don't pay too much attention on `Profile.js`